build-system/compile/bundles.config.js

```diff
@@ -112,6 +112,15 @@ exports.jsBundles = {
       includePolyfills: true,
     },
   },
+  'compiler.js': {
+    srcDir: './src/compiler/',
```

Comment:
🚲 🏚️ ing time! Do these binaries get served on the CDN?

Reply:
I believe so!